tests/test_routes.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 from app.assess.models.flag import Flag
 from app.assess.models.score import Score
+from bs4 import BeautifulSoup
```

Comment:
This is actually a really good idea, I'm annoyed I didn't think of this.  Better to use this for assertions in test as opposed to regex.